tests/integration/ha/helpers.py

```diff
@@ -514,54 +519,59 @@ async def start_and_check_continuous_writes(ops_test: OpsTest, unit_ip: str, app
         f"Index {ContinuousWrites.INDEX_NAME} has {initial_count} documents, starting there"
     )
     writer = ContinuousWrites(ops_test, app, initial_count=initial_count)
-    await writer.start()
-    time.sleep(60)
+    await writer.start(is_bulk=True)
```

Comment:
nit: `is_bulk` set to True by default. No need to specify it. We generally don't use the "non bulk" write as it's not representative of how OpenSearch is used.